src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceAfterCheck.java

```diff
@@ -103,19 +104,27 @@
             TokenTypes.DOT,
             TokenTypes.TYPECAST,
             TokenTypes.ARRAY_DECLARATOR,
-            TokenTypes.GENERIC_START,
-            TokenTypes.GENERIC_END,
+            TokenTypes.INDEX_OP,
         };
     }
 
     @Override
-    public int[] getRequiredTokens() {
-        return ArrayUtils.EMPTY_INT_ARRAY;
-    }
-
-    @Override
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The method visitToken() has an NPath complexity of 16](https://www.codacy.com/app/checkstyle/checkstyle/file/1094131143/issues/source?bid=2193633&fileBranchId=2739119#l112)